kornia/color/luv.py

```diff
@@ -0,0 +1,191 @@
+from typing import Tuple
+import torch
+import torch.nn as nn
+
+from .xyz import rgb_to_xyz, xyz_to_rgb
+
+"""
+The RGB to Luv color transformations were translated from scikit image's rgb2luv and luv2rgb
+
+https://github.com/scikit-image/scikit-image/blob/a48bf6774718c64dade4548153ae16065b595ca9/skimage/color/colorconv.py
+
+"""
+
+
+class RgbToLuv(nn.Module):
+    r"""Converts an image from RGB to Luv
+
+    The image data is assumed to be in the range of :math:`[0, 1]`. Luv
+    color is computed using the D65 illuminant and Observer 2.
+
+    args:
+        image (torch.Tensor): RGB image to be converted to Luv.
```

Comment:
make sure it outputs in the correct format, I don't see end lines

Reply:
Added